modules/restore-add-master.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-add-master_{context}"]
+= Adding a master host back to the etcd cluster
+
+Follow these steps to add a master host back to the etcd cluster. This procedure assumes that you previously removed the master host from the cluster and that its etcd dependencies are valid, such as TLS certificates and DNS.
```

Comment:
Do you mean:
s/etcd dependencies are valid, such as TLS certificates and DNS/etcd dependencies, such as TLS certificates and DNS, are valid/
?